teams/infra/pull-request/README.md

```diff
@@ -3,6 +3,31 @@ This collection of github actions is currently intended to enable `Auto-Releases
 
 If applied correctly, every commit to the mainline branch of a repository will be [added to a Release PR](./release/README.md) which then can be [automerged](./automerge/README.md) on a schedule.
 
+## Available Actions
+
+This package contains three complementary actions for pull request management:
+
+* **[Release](./release/README.md)** - Automated release PR creation and management based on conventional commits
+* **[Automerge](./automerge/README.md)** - Automated merging of release PRs on a schedule
+* **[Validate Title](./validate-title/README.md)** - Validates PR titles against conventional commit format (recommended for side projects)
+
+## PR Title Validation for Side Projects
+
+⚠️ **Important for Side Projects**: If your repository uses the release action and "Squash & Merge" strategy, it's highly recommended to also use the [validate-title](./validate-title/README.md) action. This prevents broken releases when PR titles don't follow conventional commit format.
```

Comment:
You could also mention when this happens:

This is the case when at the time of PR creation there is already more than 1 commit.
Then Github seems to imagine a title which diverges from any convention.

Reply:
also it's not only important for side projects, but all projects using release-please to manage releases.